Source/MediaInfo/File__Analyze_Streams_Finish.cpp

```diff
@@ -1137,6 +1137,10 @@ void File__Analyze::Streams_Finish_StreamOnly_General(size_t StreamPos)
         if (!CompanyName.empty() && Model.rfind(CompanyName + __T(' '), 0) == 0) {
             Fill(Stream_General, StreamPos, General_Encoded_Hardware_Model, Model.substr(CompanyName.size() + 1), true);
         }
+        if (CompanyName == __T("NIKON CORPORATION") && Model.rfind(__T("NIKON "), 0) == 0) {
+            Fill(Stream_General, StreamPos, General_Encoded_Hardware_CompanyName, "Nikon", static_cast<size_t>(-1), true, true);

```

Comment:
```suggestion
            Fill(Stream_General, StreamPos, General_Encoded_Hardware_CompanyName, "Nikon", Unlimited, true, true);
```

You were right but use the shortcut 'Unlimited' (I should do the static_cast on it, I know...) so it is as elsewhere.